webserver/src/main/java/io/kestra/webserver/controllers/ExecutionController.java

```diff
@@ -753,6 +774,34 @@ public HttpResponse<?> kill(
         return HttpResponse.noContent();
     }
 
+    @ExecuteOn(TaskExecutors.IO)
+    @Post(uri = "executions/{executionId}/resume", produces = MediaType.TEXT_JSON)
+    @Operation(tags = {"Executions"}, summary = "Resume a paused execution.")
+    @ApiResponse(responseCode = "204", description = "On success")
+    @ApiResponse(responseCode = "409", description = "if the executions is not paused")
+    public HttpResponse<?> resume(
+        @Parameter(description = "The execution id") @PathVariable String executionId
+    ) throws InternalException {
+        Optional<Execution> maybeExecution = executionRepository.findById(executionId);
+        if (maybeExecution.isEmpty()) {
+            return HttpResponse.notFound();
+        }
+
+        var execution = maybeExecution.get();
+        if (!execution.getState().isPaused()) {
+            throw new IllegalStateException("Execution is not paused, can't resume it");
+        }
+
+        var runningTaskRun = execution.findFirstByState(State.Type.PAUSED).map(taskRun ->
```

Comment:
I think we should handle multiple pause task inside an execution

Reply:
This is not currently possible if multiple tasks are paused at the same time, using Parallel or EachParallel, only the first that finishes will continue the other ones will stay paused forever. So the current resume implementation is coherent with what if currently possible for using the Pause task in a flow.

So we must fix the bug of multiple simultaneous pause first, otherwise this is not testable. I can open an issue with a reproducer and note that resume must also be fixed.